Makefile

```diff
@@ -985,10 +986,10 @@ ifeq (${CI}, true)
 ci-sweep: ## Sweep up the CI to try to get more disk space.
 	@echo "Preparing the CI for build by sweeping up disk space a bit..."
 	df -h
-	sudo apt-get --purge autoremove
-	sudo apt-get clean
-	sudo rm -rf "/opt/*" "/usr/local/*"
-	sudo rm -rf "/usr/local/share/boost" && sudo rm -rf "${AGENT_TOOLSDIRECTORY}"
+	${SUDO} apt-get --purge autoremove
```

Comment:
Why is this needed? The logic looks rather odd to me, in which cases will it kick in?

Reply:
We need to cater for two scenarios - sudo locally and no sudo inside our containers.

Reply:
Why do we run this inside the container though? We won't be clearing up space due to how docker layers work, and if this is supposed to clear the host paths from inside the container - it won't work either.